frontend/src/component/poll_form.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{
+    codegen::poll_service::{
+        ClosePollRequest, GetPollBySlugResponse, SubmitVoteRequest, VoteOption,
+    },
+    hooks::use_poll_service::use_poll_service,
+    poll_kind::poll_kind_id_to_label,
+    toast::use_toast,
+};
+use js_sys::Date;
+use prost_types::Timestamp;
+use yew::{platform::spawn_local, prelude::*};
+use yew_hooks::use_local_storage;
+
+fn current_timestamp() -> Timestamp {
+    let now = Date::new_0();
+    Timestamp::date_time_nanos(
+        now.get_utc_full_year().into(),
+        (now.get_utc_month() + 1).try_into().unwrap(),
+        now.get_utc_date().try_into().unwrap(),
+        now.get_utc_hours().try_into().unwrap(),
+        now.get_utc_minutes().try_into().unwrap(),
+        now.get_utc_seconds().try_into().unwrap(),
+        now.get_utc_milliseconds() * 1000,
```

Comment:
Shouldn't `get_utc_milliseconds * 1000` be equal to `get_utc_seconds`? 🤔

Reply:
Let's say we have a time like 13:12:11.9
We got 11 seconds and 900 millis
I don't think that `900 * 1000 == 11` in that case

Reply:
Ah, yeah, true, somehow I read it as division (i.e. if we divide 900 ms by 1000 we'll 0.9 s). But I guess here we need nanoseconds instead of milliseconds